core/src/main/java/com/linecorp/armeria/client/limit/AbstractConcurrencyLimitingClient.java

```diff
@@ -56,10 +56,9 @@
 
     private static final long DEFAULT_TIMEOUT_MILLIS = 10000L;
```

Comment:
Could you remove this static field and use the same field in `ConcurrencyLimitBuilder`, please?